fudge-ratio/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.sps</groupId>
+  <artifactId>google-cloud-datastore-todo-list</artifactId>
```

Comment:
Maybe we should come up with our own id 😊
Possible ideas: (fudge_ratio, fudge_ratio_calculator, fudge_ratio_team36)